### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by 78gldnbear** [PDConSec/vsc-print#224] [Q&A] [answered]
I just installed your extension.  When I try to print, I get the following response:

The connection for this site is not securelocalhost sent an invalid response.

[Try running Windows Network Diagnostics]
(javascript:diagnoseErrors()).

ERR_SSL_PROTOCOL_ERROR

I added VSC to apps to allow through the firewall, but that did not help.

I am running Windows 11 Home, and my browser is Microsoft Edge.

Thanks.

**Comment by PeterWone** [comment, marked as answer]
Configure your system to allow SSL for localhost. 

The Print extension does not use SSL for the embedded webserver because this would entail certificate management on user workstations for no benefit. It is normal to permit unsecured HTTP for localhost.

Occasionally this is a problem with the browser. More information and resolution strategies here:

https://www.hostitsmart.com/manage/knowledgebase/264/Fix-localhost-sent-an-invalid-response-ERR-SSL_PROTOCOL_ERROR.html
